ckanext/data_qld/resources_plugin.py

```diff
@@ -11,8 +11,11 @@
 import helpers
 import logging
 
+from ckan.common import request
 from flask import Blueprint
 from de_identified_data import helpers as de_identified_data_helpers
+from resource_visibility import helpers as resource_visibility_helpers
```

Comment:
Import helpers using full path namespace for CKAN 2.9 compatibility eg. `from ckanext.data_qld.resource_visibility import helpers`
Can you update the `de_identified_data` helper import also

Reply:
refactored